65black/StackOlite
# #160717874 Post and View Comments

#### What does this PR do?
This PR consumes the `POST /api/v1/questions/:questionId/answers/:answerId/comments` endpoint from [this repo](https://github.com/65black/StackOverflow-lite).

#### How should this be manually tested?
 - Clone this repo and open index.html with Chrome.
 - Log in to an existing account.
 - Navigate to questions page.
 - Open any question with at least one answer.
 - Click on add comment and input your comment.
 - Click on the big add comment button and the page should reload.
 - Click on show comments and notice your newly added comment.

#### Screenshots
![comments](https://user-images.githubusercontent.com/36507972/45949130-a66dba80-bff2-11e8-8e09-6427bc42ed9c.JPG)

#### What are the relevant pivotal tracker stories?
#160717874
